oaknational/Oak-Web-Application
# Feat/subject icons

## Description

add subject icons throughout teacher view pages
remove ks4 combined science tier bug

## Issue(s)

Fixes #1202, #1361  



## How to test

1. Go to beta/teachers
2. Click all theway to downloads
3. You should see correct subject icons

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
